assets/js/components/notifications/ZeroDataStateNotifications.stories.js

```diff
@@ -0,0 +1,231 @@
+		compareStartDate: '2021-08-18',
+		compareEndDate: '2021-09-14',
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Total Users',
+			},
+		],
+		dimensions: [ 'ga:date' ],
+		dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+	},
+	{
+		dimensions: [ 'ga:date' ],
+		metrics: [
+			{
+				expression: 'ga:users',
+			},
+		],
+		startDate: '2021-09-15',
+		endDate: '2021-10-12',
+	},
+];
+
+const Template = ( { setupRegistry } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<ZeroDataStateNotifications />
+	</WithRegistrySetup>
+);
+
+export const NoNotificationsAvailable = Template.bind( {} );
+NoNotificationsAvailable.storyName = 'No Notifications Available';
+NoNotificationsAvailable.args = {
+	setupRegistry: ( registry ) => {
+		provideSearchConsoleMockReport( registry, searchConsoleArgs );
+
+		for ( const options of analyticsArgs ) {
+			provideAnalyticsMockReport( registry, options );
+		}
+	},
+};
+
+export const AnalyticsGatheringData = Template.bind( {} );
+AnalyticsGatheringData.storyName = 'Analytics Gathering Data ';
+AnalyticsGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		provideSearchConsoleMockReport( registry, searchConsoleArgs );
+		for ( const options of analyticsArgs ) {
+			registry
+				.dispatch( MODULES_ANALYTICS )
+				.receiveGetReport( [], { options } );
+		}
+	},
+};
+
+export const SearchConsoleGatheringData = Template.bind( {} );
+SearchConsoleGatheringData.storyName = 'Search Console Gathering Data ';
+SearchConsoleGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		registry
+			.dispatch( MODULES_SEARCH_CONSOLE )
+			.receiveGetReport( [], { options: searchConsoleArgs } );
+		for ( const options of analyticsArgs ) {
+			provideAnalyticsMockReport( registry, options );
+		}
+	},
+};
+
+export const SearchConsoleAndAnalyticsGatheringData = Template.bind( {} );
+SearchConsoleAndAnalyticsGatheringData.storyName =
+	'Search Console And Analytics Gathering Data ';
+SearchConsoleAndAnalyticsGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		registry
+			.dispatch( MODULES_SEARCH_CONSOLE )
+			.receiveGetReport( [], { options: searchConsoleArgs } );
+		for ( const options of analyticsArgs ) {
+			registry
+				.dispatch( MODULES_ANALYTICS )
+				.receiveGetReport( [], { options } );
+		}
+	},
+};
+
+export const ZeroDataState = Template.bind( {} );
+ZeroDataState.storyName = 'Zero Data State';
+ZeroDataState.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetReport(
+			[
+				{
+					clicks: 0,
+					ctr: 0,
+					impressions: 0,
+					keys: [ '2021-08-18' ],
+					position: 0,
+				},
+			],
+			{
+				options: searchConsoleArgs,
+			}
+		);
+		for ( const options of analyticsArgs ) {
+			provideAnalyticsMockReport( registry, options );
+		}
+	},
+};
+
+export default {
+	title: 'Components/ZeroDataStateNotifications',
+	decorators: [
+		( Story ) => (
+			<div className="googlesitekit-widget">
```

Comment:
@hussain-t We don't need to wrap within the `.googlesitekit-widget` since it's not a widget. That's why when viewing the story, there's some extra white space.